hvplot/converter.py

```diff
@@ -383,6 +285,126 @@ class HoloViewsConverter:
     tiles_opts (default=None): dict
         Options to customize the tiles layer created when `tiles` is set,
         e.g. `dict(alpha=0.5)`.
+
+    Data Options
+    ------------
+    attr_labels (default=None): bool
+        Whether to use an xarray object's attributes as labels, defaults to
+        None to allow best effort without throwing a warning. Set to True
+        to see warning if the attrs can't be found, set to False to disable
+        the behavior.
+    by (default=None): str or list of str
+        Dimension(s) by which to group the data categories.
+        An NdOverlay is returned by default unless `subplots=True`, then an NdLayout is returned.
+    fields (default={}): dict
+        A dictionary of fields for renaming or transforming data dimensions.
+    groupby (default=None): str or list
+        Dimension(s) by which to group data, enabling widgets.
+        Returns a DynamicMap if `dynamic=True`, else returns a HoloMap.
+        See :paramref:`dynamic` for more information.
+    group_label (default=None): str
+        Label for grouped data, typically in legends or axis labels.
+    kind (default='line'): str
+        The type of plot to generate.
+    label (default=None): str
+        Label for the data, typically used in the plot title or legends.
+    robust: bool
+        If True and clim are absent, the colormap range is computed
+        with 2nd and 98th percentiles instead of the extreme values
+        for image elements. For RGB elements, clips the "RGB", or
+        raw reflectance values between 2nd and 98th percentiles.
+        Follows the same logic as xarray's robust option.
+    row (default=None): str
+        Column name to use for splitting the plot into separate subplots by rows.
+    col (default=None): str
+        Column name to use for splitting the plot into separate subplots by columns.
+    sort_date (default=True): bool
+        Whether to sort the x-axis by date before plotting
+    stacked (default=False): boolean
+        Whether to stack data in plots like `bar`, `barh` and `area`.
+    subplots (default=False): boolean
+        Whether to display data in separate subplots when using the `by` parameter.
```

Comment:
Same question, are these really data options?

Reply:
In fact, if `stacked` only applies to 3 types of plot, maybe it's best to document it in their docstring directly instead of globally?

Reply:
It makes sense to document `stacked` directly instead of globally but I think `subplots` fit as it's related to grouping the data in some way.